versioned_hdf5/api.py

```diff
@@ -150,3 +156,25 @@ def stage_version(self, version_name: str, prev_version=None,
         except:
             delete_version(self.f, version_name, old_current)
             raise
+
+    def close(self):
+        """
+        Make sure the VersionedHDF5File object is no longer reachable.
+        """
+        if not self._closed:
+            del self.f
+            del self._version_data
+            del self._versions
+            self._closed = True
+
+    def __repr__(self):
+        """
+        Prints friendly status information.
+
+        These messages are intended to be similar to h5py messages.
+        """
+        if self._closed:
+            return "<Closed VersionedHDF5File>"
```

Comment:
We could probably provide some of the same metadata in this string as below

Reply:
If the `VersionedHDF5File` object is already closed, it doesn't have any metada associated with it anymore (so the object file and metadata is no longer reachable). Is there anything specifically you think might be good to include in this string?

Reply:
oh ok.... nm then